### ISSUE TITLE
Link target not checked

### ISSUE THREAD

**Issue by beatngu13** [retest/recheck-web#145]
When changing the target of a link, for instance

```html
<li><span><a href="https://www.retest.de/en/for-testers/index.html">For Testers</a></span>
```

to

```html
<li><span><a href="SOME BROKEN URL">For Testers</a></span>
```

then the change is not detected. Consequently, we should also check `href`.

**Comment by beatngu13** [comment]
See PR #152.
